### PR TITLE
Switch order of scripts

### PR DESCRIPTION
**Pull request by isotuela**
Hi Karla and Erik, 
I look forward to reading your comments. 
I have finally swap the order of the scripts to execute at the beginnning of the observing night. We should start with the correct_pointing.py followed by a latiss_wep_align.py, and not the other way around, as the correct_pointing.py script resets the x and y hexapod offsets. 
Many thanks,
Ioana

### CODE REVIEW

File: Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/Center-absorbPointingOffsets-alignment-focus.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Procedure Steps
+===============
+
+- Confirm sky background is dark enough. 
+
+  As a general rule, a good time to start executing the following SAL scripts is 10-15 minutes before the end of nautical twilight. 
+
+- Command the telescope to measure and correct for any nightly pointing zero-point offsets using the SAL script ``auxtel/correct_pointing.py``.
+
+  The steps the script perform in the background are:
```
Comment (edennihy) [inline Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/Center-absorbPointingOffsets-alignment-focus.rst:47]: minor typo: perform -> performs

File: Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/Center-absorbPointingOffsets-alignment-focus.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
+.. |author| replace:: *TDB*
```
Comment (edennihy) [inline Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/Center-absorbPointingOffsets-alignment-focus.rst:1]: Don't forget to update these fields and the corresponding contact fields below.

**Review by edennihy — APPROVED** [review]
A few very minor comments, otherwise everything looks good!

**Comment by isotuela** [comment]
Thank you both! It's merged now.

**Pull request merged**